src/Installers/Windows/WindowsHostingBundle/ANCM.wxs

```diff
@@ -10,7 +11,7 @@
                         Compressed="yes"
                         Vital="yes"
                         Visible="no"
-                        InstallCondition="NOT VersionNT64 AND (VersionNT &gt;= v6.1) AND (IISCoreWebEngineInstalled_x86=1) AND (IISW3SVCInstalled_x86=1) AND (NOT OPT_NO_ANCM OR OPT_NO_ANCM=&quot;0&quot;)">
```

Comment:
Could you explain this change?

Reply:
I guess there's one difference here in that we no longer are checking for VersionNT >= 6.1, we could bring that back for all 3 of these if that prevents older windows installs?